ush/global_cycle_driver.sh

```diff
@@ -9,6 +9,7 @@ set -eux
 #-------------------------------------------------------------------------------------------------
 
 export CASE=${CASE:-C768}                    # resolution of tile: 48, 96, 192, 384, 768, 1152, 3072
+ocn=${ocn:-""}                               # ocean grid resolution: 025, 100, etc.
```

Comment:
how about 025, 050,100 and 500 instead of 025, 100, etc

Reply:
Done.